docs/docs/concepts/infrastructure-architecture.mdx

```diff
@@ -0,0 +1,90 @@
+---
+sidebar_position: 1
+---
+
+# Infrastructure Architecture
+
+JHub Apps, also known as JupyterHub Apps Launcher, is an [Externally-Managed JupyterHub service](https://jupyterhub.readthedocs.io/en/stable/reference/services.html#externally-managed-services) designed to enable users to launch various server types including API services, dashboards such as Panel, Bokeh, Streamlit, standard JupyterLab instances, and any generic Python command.
+
+The system is built on a FastAPI framework as detailed in the [services.apps module](https://github.com/nebari-dev/jhub-apps/blob/main/jhub_apps/service/app.py). It initializes and runs through a [Uvicorn](https://www.uvicorn.org/) server (a WSGI HTTP server), managed by the JupyterHub service manager upon startup:
+
+<div style={{'border-radius': '15px', 'border': '2px solid var(--ifm-color-primary)', 'padding': '20px' }}>
+```python title="jhub_apps/configuration.py#L42-L5"
+c.JupyterHub.services.extend(
+    [
+        {
+            "name": fast_api_service_name,
+            "url": f"http://{c.JAppsConfig.hub_host}:10202",
+            "command": [
+                c.JAppsConfig.python_exec,
+                "-m",
+                "uvicorn",
+                "jhub_apps.service.app:app",
+                "--port=10202",
+                "--host=0.0.0.0",
+                f"--workers={c.JAppsConfig.service_workers}",
+            ],
+            ...
+        }
+    ]
+)
+```
+</div>
```

Comment:
I'm confused on what this is - is this something jhub-apps implements on the backend on my behalf? Or is this how to configure jhub to use jhub-apps (and if so, why is it different than the [installation docs](https://jhub-apps-docs-git-fork-viniciusdc-255-3edd9d-nebari-jhub-apps.vercel.app/docs/installation))?

Reply:
when you call `install_jhub_apps` this is what it does under the hood, I added here just to show where uvicorn was called and to illustrate that jhub-apps in the perspective of jupyterhub is just a system call command

Reply:
I removed this, as it was confusing. Instead, I added a ref. link to where was called out